internal/kafka/internal/services/quota/quota_management_list_service.go

```diff
@@ -207,3 +208,9 @@ func (q QuotaManagementListService) detectBillingModel(kafka *dbapi.KafkaRequest
 func (q QuotaManagementListService) DeleteQuota(SubscriptionId string) *errors.ServiceError {
 	return nil // NOOP
 }
+
+// noop method, subscriptions do not apply to quota management list.
+// This function will always return true
+func (q QuotaManagementListService) IsSubscriptionActive(kafka *dbapi.KafkaRequest) (*bool, error) {
+	return &[]bool{true}[0], nil
```

Comment:
We have an expiration configuration in the quota list as well. Can we use that to decide if the quota has expired or not?

Reply:
oh good point! forgot about that 😃 will update this to use that.